target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -154,6 +154,7 @@ netgear,wn3000rp-v3)
 	set_wifi_led "$boardname:green:router"
 	;;
 netgear,ex3700|\
+netgear,ex6120|\
```

Comment:
Maybe You use dts method instead?
https://github.com/openwrt/openwrt/blob/a176f8d3ecf09df9d40aaff8271284699695d450/target/linux/ramips/dts/mt7620a_hiwifi_hc5761.dts#L32